packages/android-sdk-platform-tools/PKGBUILD

```diff
@@ -1,38 +1,38 @@
 # Maintainer: Zhang Hai <dreaming.in.code.zh@gmail.com>
+# Contributor: Felix Golatofski <contact@xdfr.de>
 # Contributor: Vlad M. <vlad@archlinux.net>
 # Contributor: Gordin <9ordin @t gmail dot com>
 # Contributor: Christoph Bayer <chrbayer@criby.de>
 
 pkgname=android-sdk-platform-tools
-pkgver=r29.0.5
+pkgver=30.0.3
```

Comment:
You kept the `r` pkgver for `android-sdk-build-tools` and move it here. Also I'm not sure of teh effect of changing the `pkgver` without using an epoc.